upi/openstack/down-control-plane.yaml

```diff
@@ -17,9 +17,9 @@
     with_indexed_items: "{{ [os_cp_server_name] * os_cp_nodes_number }}"
 
   - name: 'Remove the Control Plane server group'
-    os_server_group:
-      name: "{{ os_cp_server_group_name }}"
-      state: absent
+    command:
+      # os-compute-api-version 2.15 or higher is required for the 'soft-anti-affinity' policy
+      cmd: "openstack --os-compute-api-version=2.15 server group delete {{ os_cp_server_group_name }}"
```

Comment:
One last note: this is going to fail if the server group does not exist. Have you experienced errors with the Ansible module? If so, I am afraid that we might re-do the list-parse-use dance we do in the `up` counterpart

Reply:
Yes, it will fail if the server group does not exist.
```
$ ansible-playbook -i inventory.yaml down-control-plane.yaml 

PLAY [all] ************************************************************************************************

TASK [Compute resource names] *****************************************************************************
ok: [localhost]
ok: [bastion]

PLAY [localhost] ******************************************************************************************

TASK [Remove the Control Plane servers] *******************************************************************
ok: [localhost] => (item=[0, 'openshift47-cbtmc-master'])
ok: [localhost] => (item=[1, 'openshift47-cbtmc-master'])
ok: [localhost] => (item=[2, 'openshift47-cbtmc-master'])

TASK [Remove the Control Plane server group] **************************************************************
fatal: [localhost]: FAILED! => {"changed": true, "cmd": ["openstack", "--os-compute-api-version=2.15", "server", "group", "delete", "openshift47-cbtmc-master"], "delta": "0:00:04.377825", "end": "2021-01-24 01:03:52.192066", "msg": "non-zero return code", "rc": 1, "start": "2021-01-24 01:03:47.814241", "stderr": "WARNING: yacc table file version is out of date\nNo servergroup with a name or ID of 'openshift47-cbtmc-master' exists.\n1 of 1 server groups failed to delete.", "stderr_lines": ["WARNING: yacc table file version is out of date", "No servergroup with a name or ID of 'openshift47-cbtmc-master' exists.", "1 of 1 server groups failed to delete."], "stdout": "", "stdout_lines": []}

PLAY RECAP ************************************************************************************************
bastion                    : ok=1    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   
localhost                  : ok=2    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   

```